website/project/views/drafts.py

```diff
@@ -121,6 +123,12 @@ def submit_draft_for_review(auth, node, draft, *args, **kwargs):
     :rtype: dict
     :raises: HTTPError if embargo end date is invalid
     """
+    if waffle.switch_is_active(features.OSF_PREREGISTRATION):
+        raise HTTPError(http.GONE, data={
+            'message_short': 'The Prereg Challenge has ended.',
+            'message_long': 'The Prereg Challenge has ended. This endpoint is currently depreciated.'
```

Comment:
Typo: "deprecated". That said, this endpoint isn't really deprecated (I mean...all of v1 is deprecated, but that's besides the point)--it is still used for other schemas.

Maybe it makes sense for `message_long` to be `The Prereg Challenge has ended. No new submissions are accepted at this time. `message_short` can be unset, I think.

Reply:
Oh, ok. Then I suppose it's fine to deprecate this endpoint. Just need to make sure we weren't exposing language like "deprecated endpoint" in the UI.